controllers/admin/AdminMollieAjaxController.php

```diff
@@ -33,5 +37,16 @@ public function postProcess()
             ));
         }
 
+        if ($action === 'resendPaymentMail') {
+            $orderId = Tools::getValue('id_order');
+
+            /** @var MolliePaymentMailService $molliePaymentMailService */
+            $molliePaymentMailService = $this->module->getContainer(MolliePaymentMailService::class);
+
+            $response = $molliePaymentMailService->sendSecondChanceMail($orderId);
+
+            $this->ajaxDie(json_encode($response));
+        }
+
```

Comment:
you can refactor this controller and use something like this:

```php
class MyClass {

   public function init() {
     $action = Tools::getValue('action');

    $this->{$action}();
   }

  private function action1() {},
 private function action2() {}
}
```
the benefit would be that you will avoid if conditions and at the same time will separate logic to different methods